components/Navbar/NavbarTabletMobile/MenuTabletMobile.jsx

```diff
@@ -74,7 +74,7 @@ const LoggedOutMenu = injectIntl(({ intl: { formatMessage }, user, visible }) =>
   );
 });
 const UniverseHome = () => (
-  <Menu.Item href="/">
+  <Menu.Item>
```

Comment:
does this mean clicking the logo no longer goes to the homepage?

Reply:
Yes, i believe arthur said we shouldn't be going to the homepage on logo click

Reply:
weird... I expect the logo to work like that on every website. Oh well